fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVPartitionInfo.java

```diff
@@ -17,28 +17,41 @@
 
 package org.apache.doris.mtmv;
 
+import org.apache.doris.analysis.Expr;
+import org.apache.doris.catalog.Column;
 import org.apache.doris.common.AnalysisException;
 
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.ImmutableSortedSet;
 import com.google.gson.annotations.SerializedName;
 
+import java.util.List;
+
 /**
  * MTMVPartitionInfo
  */
 public class MTMVPartitionInfo {
 
     public enum MTMVPartitionType {
         FOLLOW_BASE_TABLE,
+        EXPR,
```

Comment:
ROLL_UP is better?

Reply:
may be have other expr,such as `str_to_date`